ext/web/lib.deno_web.d.ts

```diff
@@ -889,3 +889,22 @@ declare class DecompressionStream {
   readonly readable: ReadableStream<Uint8Array>;
   readonly writable: WritableStream<Uint8Array>;
 }
+
+/** Dispatch an uncaught exception. Similar to:
+ * ```ts
+ * setTimeout(() => { throw error; }, 0);
+ * ```
+ * The error can not be caught with a `try/catch` block. An error event will
+ * be dispatched to the global scope. You can prevent the error from being
+ * reported to the console with `Event.prototype.preventDefault()`:
+ * ```ts
+ * addEventListener("error", (event) => {
+ *   event.preventDefault();
+ * });
+ * reportError(new Error("foo")); // Will not be reported.
+ * ```
+ * In Deno, this error will terminate the process if not intercepted like above.
+ */
+declare function reportError(
+  error: Error,
```

Comment:
[TypeScript types this as `any`](https://github.com/microsoft/TypeScript/blob/e64f04bd8c87ece7f800d10d8eb90b7e592907ad/src/lib/webworker.generated.d.ts#L5188)

Reply:
Ah, I was 99% sure it would be `any` but couldn't find that and went with the forward-compatible option for now. There it is.